src/tools/build-manifest/src/main.rs

```diff
@@ -392,6 +393,10 @@ impl Builder {
             // conflicts with the rust-docs component when installed. See
             // #75833.
             // self.extend_profile("complete", &mut manifest.profiles, &["rustc-docs"]);
+
+            // JSON docs are not yet stable, therefore we only include them in the nightly complete
+            // profile for the time being
+            self.extend_profile("complete", &mut manifest.profiles, &["rust-json-docs"]);
```

Comment:
I wouldn't include them in the complete profile, nightly or not. In general the complete profile isn't really something we need to extend further IMO.